setup.py

```diff
@@ -72,6 +71,13 @@
             'codecov',
 
             'pytest-cov',
+        ],
+        ':python_version<"3.3"': [
+            # functools_lru_cache has been added to stdlib in Python 3.2
+            # however it's only got support for ``typed`` arg in Python 3.3
+            'backports.functools_lru_cache',
+        ],
+        'testing:python_version<"3.3"': [
```

Comment:
I've rejected this change before, because backports.unittest_mock is intended to be unconditionally included and it [provides the conditionality](https://github.com/jaraco/backports.unittest_mock/blob/2fa7e1bd03ba7dfdea2bbbc72419ad915508607e/setup.py#L37-L42) (only including mock on the platforms where needed). In addition to making these dependency declarations more complicated in this project, it also implies that this is the rule that should be copied into every project that uses it. I reject that notion in that it violates DRY and adds to the cognitive burden of every project that uses it.

Reply:
Are you saying that we should revert it?